python/paddle/nn/functional/common.py

```diff
@@ -2081,67 +2082,64 @@ class centers and the shape of sampled_class_center will be [num_positive_class_
     .. code-block:: python
         :name: code-example1
 
-        # CPU or single GPU
-        import paddle
-        num_classes = 20
-        batch_size = 10
-        num_samples = 6
-        label = paddle.randint(low=0, high=num_classes, shape=[batch_size], dtype='int64')
-        remapped_label, sampled_class_index = paddle.nn.functional.class_center_sample(label, num_classes, num_samples)
-
-        print(label)
-        print(remapped_label)
-        print(sampled_class_index)
-
-        # the output is
-        #Tensor(shape=[10], dtype=int64, place=CPUPlace, stop_gradient=True,
-        #       [11, 5 , 1 , 3 , 12, 2 , 15, 19, 18, 19])
-        #Tensor(shape=[10], dtype=int64, place=CPUPlace, stop_gradient=True,
-        #       [4, 3, 0, 2, 5, 1, 6, 8, 7, 8])
-        #Tensor(shape=[9], dtype=int64, place=CPUPlace, stop_gradient=True,
-        #       [1 , 2 , 3 , 5 , 11, 12, 15, 18, 19])
+        >>> # CPU or single GPU
+        >>> import paddle
+        >>> num_classes = 20
+        >>> batch_size = 10
+        >>> num_samples = 6
+        >>> label = paddle.randint(low=0, high=num_classes, shape=[batch_size], dtype='int64')
+        >>> remapped_label, sampled_class_index = paddle.nn.functional.class_center_sample(label, num_classes, num_samples)
+
+        >>> print(label)
+        >>> print(remapped_label)
+        >>> print(sampled_class_index)
+        >>> # the output is
+        Tensor(shape=[10], dtype=int64, place=Place(cpu), stop_gradient=True,
+        [10, 3 , 11, 14, 15, 15, 7 , 12, 5 , 13])
+        Tensor(shape=[10], dtype=int64, place=Place(cpu), stop_gradient=True,
+        [3, 0, 4, 7, 8, 8, 2, 5, 1, 6])
+        Tensor(shape=[9], dtype=int64, place=Place(cpu), stop_gradient=True,
+        [3 , 5 , 7 , 10, 11, 12, 13, 14, 15])
```

Comment:
这个形式看起来不太整洁，要不就一行print一行输出（输出的结果务必按照seed结果同步修改），如下：
```suggestion
        >>> print(label)
        Tensor(shape=[10], dtype=int64, place=Place(cpu), stop_gradient=True,
        [10, 3 , 11, 14, 15, 15, 7 , 12, 5 , 13])
        
        >>> print(remapped_label)
        Tensor(shape=[10], dtype=int64, place=Place(cpu), stop_gradient=True,
        [3, 0, 4, 7, 8, 8, 2, 5, 1, 6])
        
        >>> print(sampled_class_index)
        Tensor(shape=[9], dtype=int64, place=Place(cpu), stop_gradient=True,
        [3 , 5 , 7 , 10, 11, 12, 13, 14, 15])
```